### PR TITLE
Update DIAData Space

### PR DESCRIPTION
**Pull request by bitsandtea**
Update our space to include:

- New strategy
- new css
- invalidate a proposal

Etherscan: https://etherscan.io/token/0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419
CoinGecko: https://www.coingecko.com/en/coins/dia
Website: https://diadata.org
Twitter: https://twitter.com/DIAdata_org

### CODE REVIEW

File: spaces/diadata/index.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "members": [
+    "0x5737bf56559Dd32D8880Cfa236fE613E2162e157",
+    "0x8f6Ce1D7D9f01193e5E077Ac5e2c76A6Ec6FbF0C"
+  ],
+  "filters": {
+    "defaultTab": "core",
+    "minScore": 10000000,
+    "onlyMembers" : true,
+    "invalids": [ // List of proposals IDs (use this to remove a proposal from your space)
```
Comment (gokaiorg) [inline spaces/diadata/index.json:27]: JSON standard does not allow comments. Please remove them.

File: spaces/diadata/index.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "token": "0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419",
```
Comment (gokaiorg) [inline spaces/diadata/index.json:2]: Please remove this field, we don't use it anymore.

File: skins/diadata.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    --primary-color: #4f33ff;
+    --bg-color: #141414;
+    --text-color: #CCE0F4;
+    --link-color: #ff7096;
+    --heading-color: #CCE0F4;
+    --border-color: #00CCB7;
+  }
+  .dforce, .diadata {
+    --block-bg: transparent;
+  }
```
Comment (gokaiorg) [inline skins/diadata.scss:14]: Please use one space per css file.
```suggestion
.diadata {
    --primary-color: #4f33ff;
    --bg-color: #141414;
    --text-color: #CCE0F4;
    --link-color: #ff7096;
    --heading-color: #CCE0F4;
    --border-color: #00CCB7;
  }
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/diadata/index.json (hunk 2 above)
Comment (bitsandtea) [reply spaces/diadata/index.json]: Done!

File: spaces/diadata/index.json (hunk 1 above)
Comment (bitsandtea) [reply spaces/diadata/index.json]: Done!

File: skins/diadata.scss (hunk 3 above)
Comment (bitsandtea) [reply skins/diadata.scss]: Done!

**Comment by gokaiorg** [comment]
There is a conflict in your PR, can you resolve it?

**Comment by bitsandtea** [comment]
> There is a conflict in your PR, can you resolve it?

Yes, just did. Thanks

File: spaces/domains.json [hunk 4]
```diff
@@ -31,6 +31,8 @@
   "vote.streamr.network": "streamr",
   "governance.etgproject.org": "etgfinance",
   "gov.pokt.network": "pokt-network",
+  "vote.niceee.org": "nice",
+  "vote.diadata.org": "diadata"
   "vote.pooltogether.com": "pooltogether",
   "vote.niceee.org": "nice",
   "vote.diadata.org": "diadata",
```
Comment (gokaiorg) [inline spaces/domains.json:38]: You have added your domain twice and a comma is missing. Please correct this file.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Related to https://github.com/snapshot-labs/snapshot.js/pull/28

File: spaces/domains.json (hunk 4 above)
Comment (bitsandtea) [reply spaces/domains.json]: Arghh, sorry for that. Fixed.

File: spaces/diadata/index.json [hunk 5]
```diff
@@ -6,11 +6,12 @@
...
     {
-      "name": "erc20-balance-of",
+      "name": "erc20-balance-of-minimum",
       "params": {
         "address": "0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419",
-        "symbol": "DIA",
-        "decimals": 18
+          "symbol": "DIA",
+          "decimals": 18,
+          "minTokenBalance": 10
```
Comment (bonustrack) [inline spaces/diadata/index.json:14]: I've done some change so you can use an existing strategy 'erc20-with-balance' using 'minBalance' see an example of the params here: https://github.com/snapshot-labs/snapshot.js/commit/6cbeec432c85aaed002d3543b5335983b7b0a58b#diff-1b836984505debc8fc7aa0ae1af07c1364c6ba9cf70060d53c2c2aaab096dfbcR28-R34

**Review by bitsandtea — COMMENTED** [review]
Yes that works

File: spaces/diadata/index.json (hunk 5 above)
Comment (gokaiorg) [inline spaces/diadata/index.json:14]:
```suggestion
          "minBalance": 10
```

File: spaces/diadata/index.json [hunk 6]
```diff
@@ -6,11 +6,12 @@
   "skin": "diadata",
   "strategies": [
     {
-      "name": "erc20-balance-of",
+      "name": "erc20-balance-of-minimum",
```
Comment (gokaiorg) [inline spaces/diadata/index.json:9]: `erc20-balance-of-minimum` is not an existing strategy. You could use an existing one https://github.com/snapshot-labs/snapshot.js/tree/master/src/strategies
```suggestion
      "name": "erc20-with-balance",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
You should not add `spaces/domains.json` in your PR.

**Comment by bitsandtea** [comment]
@gokaiorg I just accepted your commits for the strategy name. In terms of not adding the domains file I am not sure what you mean? There is a requirement to add [it here](https://docs.snapshot.page/guides/custom-domain).

We have a PR coming out this Monday so we would really appreciate if we have this approved by then. Thanks!

**Comment by gokaiorg** [comment]
> @gokaiorg I just accepted your commits for the strategy name. In terms of not adding the domains file I am not sure what you mean? There is a requirement to add [it here](https://docs.snapshot.page/guides/custom-domain).
> 
> We have a PR coming out this Monday so we would really appreciate if we have this approved by then. Thanks!

Yes you are right but you already add it and it's live. You are committing it again without changes. https://github.com/snapshot-labs/snapshot-spaces/blob/master/spaces/domains.json#L36

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**